apispec/psc-delete-delta-spec.yml

```diff
@@ -31,6 +31,17 @@ components:
             - DELETE
         delta_at:
           type: string
+        psc_kind:
```

Comment:
To keep it consistent with the upsert delta, this should probably be `kind`

Reply:
name of field changed